meinberlin/assets/scss/components/_header.scss

```diff
@@ -43,12 +47,34 @@
 
 .main-nav__logo {
     img {
-        width: 7em;
-        margin-top: -3em;
-        margin-bottom: -3em;
+        width: 6em;
+        margin-top: -2em;
+        margin-bottom: -2em;
```

Comment:
The idea here was to vertically center the logo: 1em (line-height) + 3em + 3em = 7em. So if the total height is reduced to 6em, the magins should be 2.5em.